committee-steering/governance/sig-governance.md

```diff
@@ -13,20 +13,13 @@ December
 - Keep up-to-date meeting notes, linked from the SIG's page in the community
 repo
 - Record meetings and make them publicly available on the
-[Kubernetes Community YouTube playlist]
-- Report activity with the community via the kubernetes-dev mailing list at
-least once a quarter. Whichever format the SIG uses should _always_ be reported
-to the kubernetes-dev mailing list as this is the list the community depends on
-for SIG updates.  
-  - Each SIG is assigned an update during the monthly community meeting
-  throughout the year. The meeting host will publish the notes to the
+[Kubernetes Community YouTube playlist]  
+- Report activity with the community via the kubernetes-dev@ mailing list at
+least once a year. 
+  - Each SIG is assigned an update during the [monthly community meeting]
+  throughout the year from sig-contributor-experience. The meeting host will publish the notes to the
   kubernetes-dev mailing list with the update.  
-  - Due to limited opportunities (twice a year) at the community meeting to
-  present, your SIG will need to figure out two other options to deliver on SIG
-  updates to meet the quarterly reporting requirement:
-    a. During KubeCon + CloudNativeCon or  
-    b. A presentation or summary update sent to the kubernetes-dev
-  mailing list.
```

Comment:
Maybe we make this twice a year instead of quarterly/once a year?

This would mean:
1. One community meeting update, notes published to k-dev by the meeting host
2. SIGs send a summary or presentation update to the k-dev mailing list 

While this doesn't happen today, this seems like a good middle ground to me....thoughts?

Reply:
we will need to give them a schedule of when each is due or else its not going to happen. when we were smaller and could do quarterly, there was the prompt from the community meeting host that they were due. @castrojo can we make work by adding them to the schedule as 'k-dev'.

Reply:
> when we were smaller and could do quarterly, there was the prompt from the community meeting host that they were due. 

Was this just a couple of years ago? Seems so long ago haha...

Lgtm to do this in a follow-up, please don't treat my comment as a blocker. :+1: